src/main/java/stsjorbsmod/cards/wanderer/materialcomponents/Chamomile.java

```diff
@@ -39,7 +41,6 @@ public void upgrade() {
         if (!upgraded) {
             upgradeName();
             EphemeralField.ephemeral.set(this, false);
```

Comment:
This is inconsistent with the new upgrade description (I think the intent is to match how you have the description, ie, we should leave ephemeral and exhaust as-is during the upgrade)